apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx

```diff
@@ -200,7 +203,11 @@ const FinalStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep, on
 									</ul>
 									<p>
 										{ __( 'Questions?', 'full-site-editing' ) }{ ' ' }
-										<Button isLink href="https://wordpress.com/help/contact" target="_blank">
+										<Button
+											isLink
+											href={ localizeUrl( 'https://wordpress.com/help/contact', locale ) }
```

Comment:
#48033 mentions that the original change in this PR (with the third argument) was necessary because (apparently) depending on whether a new user creates a new site, we need different support URLs. A logged-in user can use the Calypso support form while a logged-out user needs to go to the support site. Therefore, it's important here to pass in the state of the user.
```suggestion
											href={ localizeUrl( 'https://wordpress.com/help/contact', locale, undefined !== window.currentUser ) }
```

Reply:
> A logged-in user can use the Calypso support form while a logged-out user needs to go to the support site. 

While the logged-out user scenario applies for some landing pages and signup flows in Calypso, this part of the application is loaded only for an existing unlaunched wpcom site (simple or atomic) with ETK plugin and accessible only by logged-in users from the block editor.

Reply:
Ok, just wanted to make sure because I built #48645 to accommodate the logged in/logged out scenario that was requested for this PR. If that assumption is not valid, then all good.